client/firebase.js

```diff
@@ -0,0 +1,10 @@
+import firebase from 'firebase/app';
+import 'firebase/auth';
+import { firebaseConfig } from '../keys';
+
+// moved firebase's config obj to kingdom_keys in order to group it with firestore's api key
+export const app = firebase.initializeApp(firebaseConfig);
+
+// auth is used in client/AuthContext.js
+export const auth = app.auth();
+
```

Comment:
Probably belongs in a subdirectory as well

We can also directly add `firebaseConfig` here if it's for UI side only

Reply:
Went with the easier option and added it to `firebase.js`

Reply:
I added firebase to it's own directory. 
![image](https://user-images.githubusercontent.com/35232697/105070380-ce497d80-5a48-11eb-874c-ac2692693b6a.png)